server/Controller/tokenController.js

```diff
@@ -0,0 +1,61 @@
+const jwt = require('jsonwebtoken')
+const User = require('../models/user')
+
+const postTokenController = async (req, res) => {
+  const refreshToken = req.body.refreshToken
+
+  /* use the stored secret to validate the token */
+  const decodedRefreshToken = jwt.verify(
+    refreshToken,
+    process.env.REFRESH_TOKEN_SECRET
+  )
+
+  if (!decodedRefreshToken.id) {
+    return response.status(401).json({
+      error: 'Refresh token missing or invalid'
```

Comment:
This is just a note and note a request for change but for security reasons it is better not to tell the user that `Refresh token missing or invalid`. It is better to just say `unathorized` which is the standard message for `401.`